test/unit/strval_test.js

```diff
@@ -0,0 +1,59 @@
+const xdr = SorobanClient.xdr;
+const ScInt = SorobanClient.ScInt; // shorthand
+const [scValToNative, nativeToScVal] = [
+  SorobanClient.scValToNative,
+  SorobanClient.nativeToScVal
+];
+
+describe('parsing and building ScVals', function () {
+  
+
+  it('Can parse entries', function () {
+      CUSTOM_TYPES_XDR.forEach((entry, i) => {
+          console.log(entry)
+          console.log(xdr.ScSpecEntry.fromXDR(entry, 'base64'));
+      });
+      
+  });
+
+});
+
+
+function custom_types_spec() {
+  return SorobanClient.ContractSpec(xdr.Sp)
+}
+
+const CUSTOM_TYPES_XDR = ["AAAAAQAAAC9UaGlzIGlzIGZyb20gdGhlIHJ1c3QgZG9jIGFib3ZlIHRoZSBzdHJ1Y3QgVGVzdAAAAAAAAAAABFRlc3QAAAADAAAAAAAAAAFhAAAAAAAABAAAAAAAAAABYgAAAAAAAAEAAAAAAAAAAWMAAAAAAAAR",
```

Comment:
These might be easier to maintain if we have the entry as a JS object, and serialize it in the test. That would mean fewer opaque strings to maintain here if/when the XDR changes.

Reply:
Yeah this was me just trying to get a valid XDR string.